cf-java-logging-support-logback/src/test/java/com/sap/hcp/cf/logging/common/TestCustomFields.java

```diff
@@ -14,101 +13,88 @@
 
 public class TestCustomFields extends AbstractTest {
```

Comment:
I also added a test if there are 3 custom fields, but only the 1st and the 3rd are written in a log,
to check if the numbers are assigned in a consistent manner.

Reply:
Correct assignment of the numbers is tested explicitly. I will add a test to check, that unregistered fields are not added as custom fields though.